_drafts/wework-com-is-going-static.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout:       post
+title:        WeWork.com is Going Static
```

Comment:
Do we want to do a Part 1/Part 2 thing with the static react stuff?

Also do we want to be more SEO friendly with the title, like "Why WeWork.com uses a static generator and why you should too" or something including the words "why", "tutorial", "static generator", etc...